extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -812,85 +798,10 @@ class AmpLightbox extends AMP.BaseElement {
    */
   update_(pos) {
     dev().fine(TAG, 'update_');
-    this.updateChildrenInViewport_(pos, this.oldPos_);
     this.oldPos_ = pos;
     this.pos_ = pos;
   }
 
-  /**
-   * Update the inViewport status of children when scroll position changed.
-   * @param {number} newPos
-   * @param {number} oldPos
-   * @private
-   */
-  updateChildrenInViewport_(newPos, oldPos) {
-    const seen = [];
-    this.forEachVisibleChild_(newPos, (cell) => {
-      seen.push(cell);
-      const owners = Services.ownersForDoc(this.element);
-      owners.updateInViewport(this.element, cell, true);
-      owners.scheduleLayout(this.element, cell);
```

Comment:
Similar. This will require `inob-loader` launch complete, no?

Reply:
Argh. I read the jsdoc and didn't realize this also schedules layout!

Reply:
Yeah. Let's do. But this is good to know. Maybe we'll priorities loader inob polyfilling higher.